src/coreclr/vm/appdomain.cpp

```diff
@@ -2111,10 +2115,12 @@ AppDomain::AppDomain()
     m_pRootAssembly = NULL;
 
     m_dwFlags = 0;
+#if  defined(FEATURE_COMINTEROP) || defined(FEATURE_COMWRAPPERS)
 #ifdef FEATURE_COMINTEROP
     m_pRCWCache = NULL;
+#endif    
     m_pRCWRefCache = NULL;
```

Comment:
Can't we have each under their own `#ifdef`? Is there a need for the `#if defined(FEATURE_COMINTEROP) || defined(FEATURE_COMWRAPPERS)`?

Reply:
Ok. I think the or condition is needed to include m_pRCWRefCache also when FEATURE_COMWRAPPERS is enabled

Reply:
I believe that `m_pRCWRefCache` is only needed when `FEATURE_COMWRAPPERS` is set. On Windows they are both set but this member should only be used by the `ComWrappers` API.

Reply:
Ok, I removed all references to rcwrefcache from feature_comiterop and kept them only for feature_comwrappers